### DISCUSSION TITLE
Image loading

### DISCUSSION THREAD

**Discussion by yuenxillar** [emilk/egui#4983] [Q&A]
![image](https://github.com/user-attachments/assets/eab0ef80-6650-4de7-9e5e-f94cb9422802)
How to add all the image files in a folder at once instead of using this macro

**Comment by YgorSouza** [comment]
This macro is just a wrapper around `include_bytes`, so it only handles individual files. To embed an entire folder, you'd have to use something like [rust-embed](https://crates.io/crates/rust-embed).
